nifi-docker/dockerhub/sh/start.sh

```diff
@@ -97,6 +97,13 @@ prop_replace 'nifi.analytics.connection.model.implementation'   "${NIFI_ANALYTIC
 prop_replace 'nifi.analytics.connection.model.score.name'       "${NIFI_ANALYTICS_MODEL_SCORE_NAME:-rSquared}"
 prop_replace 'nifi.analytics.connection.model.score.threshold'  "${NIFI_ANALYTICS_MODEL_SCORE_THRESHOLD:-.90}"
 
+# Add NAR provider properties
+# nifi-registry NAR provider
+if [ -n "${NIFI_NAR_LIBRARY_PROVIDER_NIFI_REGISTRY_URL}" ]; then
+    prop_add 'nifi.nar.library.provider.nifi-registry.implementation' 'org.apache.nifi.registry.extension.NiFiRegistryNarProvider'
```

Comment:
The `NiFiRegistryNarProvider` was deprecated in favor of `NiFiRegistryExternalResourceProvider` as of NiFi 1.17.0 as part of [NIFI-9615](https://issues.apache.org/jira/browse/NIFI-9615). It looks like this could be replaced with the new class.
```suggestion
    prop_add 'nifi.nar.library.provider.nifi-registry.implementation' 'org.apache.nifi.registry.extension.NiFiRegistryExternalResourceProvider'
```